app/controllers/airlines_controller.rb

```diff
@@ -0,0 +1,6 @@
+class AirlinesController < ApplicationController
+  def show
+    @airline = Airline.find(params[:id])
+    @passengers = @airline.passengers.where('age >= ?', 18).uniq
+  end
```

Comment:
Doing a where in the controller is considered an MVC infraction. `.uniq` is also a ruby method that you are using here to process AR data, there is an AR equivalent that you should be using.